css/mail.scss

```diff
@@ -318,31 +318,18 @@
 }
 
 /* icons */
-.icon-inbox {
-	@include icon-color('inbox', 'mail', $color-black);
-}
-.icon-important {
-	@include icon-color('important', 'mail', $color-black);
-}
+@include icon-black-white('inbox', 'mail', 1);
+@include icon-black-white('important', 'mail', 1);
+
 .icon-flagged {
 	@include icon-color('star', 'mail', $color-black);
 }
```

Comment:
this one doesnt work as `@include icon-black-white('star', 'mail', 1);` is it maybe because in some places it is referred as flagged and in come as star? @skjnldsv ?